apis/stackgres/v1/sgpostresconfig.go

```diff
@@ -0,0 +1,49 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// This CRD isn't autogenerated, as the generator from the CRD somewhat falls apart with this one.
+// As it only has two fields it should not be too much trouble.
```

Comment:
I think we should be able to make this work, but I also struggle to do so right now.  I think this is fine for now, but maybe we should think about investing some time to un-YOLO-fy this CRD generation in the future.

Reply:
Yeah we'll have to look into this in the future.